client/components/social-logo/README.md

```diff
@@ -1,12 +1,12 @@
 Social Logo
 ========
 
-Component that renders a single socail-logo svg based on an `icon` prop. It takes a size property but defaults to 24px. For greater sharpness, the icons should only be shown at either 18, 24, 36 or 48px. There's a gallery with all the available icons in devdocs/design.
```

Comment:
Sounds good 👍  I'll try to follow up in a later PR to show examples in devdocs directly from the package vs here.